dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -214,6 +214,106 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/authoring/analyze-conversations/projects/{projectName}/:authorize-copy": {
+      "post": {
+        "description": "Generates a copy project operation authorization to the current target Azure resource.",
+        "operationId": "ConversationalAnalysisAuthoring_CopyProjectAuthorization",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The copy project authorization info.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringCopyAuthorizationOptions"
+            }
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Payload to send to source resource to initiate project copying.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringCopyProjectOptions"
```

Comment:
Why are you sending the input model back? If you're just echoing content, seems like this should be a resource `PUT`.